.travis.yml

```diff
@@ -134,6 +134,12 @@ jobs:
       go: $TRAVIS_GO_VERSION
       stage: test
 
+    # Docker Log Driver
+    - os: linux
+      env: TARGETS="-C x-pack/dockerlogbeat unit-tests"
```

Comment:
Should we have a `testsuite` target for consistency with other beats? (in this case it will only depend on `unit-tests` target)

Reply:
I tired to do that, but `testsuite` is a common target imported from the libbeat mage file, and it doesn't seem to work with the docker plugin's wierd design.

Reply:
Right now we just have unit tests, and the testsuite also wants integration tests and an `update` command.